features/supplier/supplier_signs_framework_agreement.feature

```diff
@@ -0,0 +1,24 @@
+@supplier @supplier_account @esignature @supplier_esignature
```

Comment:
how is @esignature and @supplier_esignature different?

Reply:
I guess we could use `@esignature` e.g. if we added tests for the admin side, where `@supplier_esignature` is specific to the supplier side.

Reply:
ah, thanks. I think what you have right now is good.

Reply:
Yep, looks good - no harm in having an extra granular tag